client/src/main/proto/spine/base/command.proto

```diff
@@ -77,6 +78,14 @@ message CommandContext {
 
     // The `namespace` attribute must be defined for commands in multitenant applications.
     Namespace namespace = 5;
+
+    oneof time_opts {
```

Comment:
Isn't it `scheduling_options`? Or, `schedule_by`, or just `schedule`?

How do they name such things on existing messaging systems?